src/main/scala/com/gu/appstoreconnectapi/Conversion.scala

```diff
@@ -2,20 +2,21 @@ package com.gu.appstoreconnectapi
 
 import java.time.ZonedDateTime
 
-import com.gu.appstoreconnectapi.AppStoreConnectApi.BuildsResponse
+import com.gu.appstoreconnectapi.AppStoreConnectApi.{AppStoreVersionsResponse, BuildsResponse}
 
 import scala.util.{Failure, Success, Try}
 
 object Conversion {
 
   case class LiveAppBeta(version: String, buildId: String, uploadedDate: ZonedDateTime, internalBuildState: String, externalBuildState: String)
+  case class LiveAppProduction(versionString: String, version: String)
```

Comment:
whats the difference between these fields?

Reply:
Pretty confusing eh!?

If you consider the build: `8.38-17913`.

`versionString` is the `8.38` and `version` is the longer build number `17913` (which comes from CI in our case).